modules/nodes-secondary-scheduler-configuring-console.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-secondary-scheduler-configuring-console_{context}"]
+= Deploying a secondary scheduler
+
+After you have installed the {secondary-scheduler-operator}, you can deploy a secondary scheduler.
+
+.Prerequisities
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* The {secondary-scheduler-operator-full} is installed.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Create config map to hold the configuration for the secondary scheduler.
+.. Navigate to *Workloads* -> *ConfigMaps*.
+.. Click *Create ConfigMap*.
+.. In the YAML editor, enter the config map definition that contains the necessary `KubeSchedulerConfiguration` configuration:
```

Comment:
Content of the `KubeSchedulerConfiguration` configuration is solely up to the user. The one mentioned in the docs is an example. Thus, worth mentioning "for example" in the sentence. E.g. "enter the config map definition that contains the necessary `KubeSchedulerConfiguration` configuration. For example:"